Error while running vue service with docker

`sh: vue-cli-service: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! web@0.1.0 serve: `vue-cli-service serve`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the web@0.1.0 serve script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?`

--- a/web/Dockerfile
+++ b/web/Dockerfile
@@ -2,6 +2,7 @@ FROM node:14.17.0-alpine
 WORKDIR /app/web
 COPY package.json ./
 COPY package-lock.json ./
+RUN npm install -g @vue/cli
 RUN npm install
 COPY . /app/web/
 

--- a/web/package.json
+++ b/web/package.json
@@ -8,7 +8,6 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@vue/cli": "^4.5.13",
     "axios": "^0.21.1",
     "core-js": "^3.6.5",
     "vue": "^2.6.11",

